src/AutoUpdater/AutoUpdater.h

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (C) 2008 Remko Troncon
+ * Copyright (C) 2001-2019  Psi Team
```

Comment:
This file got real code modifications in 2015. 
2019 is not true. I guess the same for many other files.

Reply:
It is not a problem.
And I have edited this file in 2019 for this perfect change :)